changelog/db.changelog-1.1.xml

```diff
@@ -0,0 +1,131 @@
+      <column name="schedule" type="VARCHAR(3)" />
+      <column name="sitiAgriInvoiceNumber" type="VARCHAR(20)" />
+      <column name="submitted" type="TIMESTAMP" />
+      <column name="value" type="INTEGER" />
+    </createTable>
+  </changeSet>
+  <changeSet author="Marc Templeton" id="4">
+    <createTable tableName="settlements">
+      <column name="settlementId" type="INTEGER" autoIncrement="true">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="settlementsId_pkey" />
+      </column>
+      <column name="paymentRequestId" type="INTEGER" >
+        <constraints nullable="false" />
+      </column>
+      <column name="detail" type="VARCHAR(255)" />
+      <column name="invalid" type="BOOLEAN" defaultValue="false" />
+      <column name="generated" type="BOOLEAN" defaultValue="false" />
+      <column name="frn" type="BIGINT" />
+      <column name="invoiceNumber" type="VARCHAR(30)" />
+      <column name="ledger" type="VARCHAR(2)" />
+      <column name="reference" type="VARCHAR(9)" /> <!-- based on PY1234567 -->
+      <column name="settled" type="BOOLEAN" defaultValue="false" />
+      <column name="settlementDate" type="TIMESTAMP" /> <!-- interesting in pay-pr this (named settled) is used for above bool -->
+      <column name="sourceSystem" type="VARCHAR(50)" />
+    </createTable>
+  </changeSet>
+  <changeSet author="Marc Templeton" id="5">
+    <addForeignKeyConstraint baseColumnNames="paymentRequestId" baseTableName="settlements" constraintName="fk_paymentRequests_paymentRequestId" referencedColumnNames="paymentRequestId" referencedTableName="paymentRequests" />
+  </changeSet>
+  <changeSet author="Marc Templeton" id="6">
+    <createTable tableName="schedules">
+      <column name="scheduleId" type="INTEGER" autoIncrement="true">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="schedulesId_pkey" />
+      </column>
+      <column name="settlementId" type="INTEGER" >
+        <constraints nullable="false" />
+      </column>
+      <column name="completed" type="TIMESTAMP" />
+      <column name="planned" type="TIMESTAMP" />
+      <column name="started" type="TIMESTAMP" />
+    </createTable>
+  </changeSet>
+  <changeSet author="Marc Templeton" id="7">
+    <addForeignKeyConstraint baseColumnNames="settlementId" baseTableName="schedules" constraintName="fk_settlements_settlementId" referencedColumnNames="settlementId" referencedTableName="settlements" />
+  </changeSet>
+  <changeSet author="Marc Templeton" id="8">
+    <createTable tableName="organisations">
+      <column name="sbi" type="INTEGER">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="organisationsId_pkey" />
+      </column>
+      <column name="address" type="VARCHAR(255)" />
+      <column name="emailAddress" type="VARCHAR(127)" />
+      <column name="frn" type="BIGINT" />
+      <column name="name" type="VARCHAR(127)" />
```

Comment:
These were all defined by me, do they seem sensible?

Reply:
Yes I think so.  Only doubt is whether address should be a single entry or whether we should have separate columns for address line 1, 2, 3, city, county, post code.  I think that would make the data easier to work with when mapping it to a PDF.